.pre-commit-config.yaml

```diff
@@ -0,0 +1,40 @@
+---
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.4.0
+    hooks:
+      - id: end-of-file-fixer
+      - id: trailing-whitespace
+      - id: mixed-line-ending
+      - id: check-byte-order-marker
+      - id: check-executables-have-shebangs
+      - id: check-merge-conflict
+      - id: debug-statements
+      - id: check-json
+      - id: check-yaml
+        files: .*\.(yaml|yml)$
+        args: ["--unsafe"]
+  - repo: https://github.com/PyCQA/flake8.git
+    rev: 3.8.3
+    hooks:
+      - id: flake8
```

Comment:
Why do we need flake8? Do we have python code in install_yamls?

Reply:
No.. we don't have python code yet... and i think we can add it later.. if we have python code.